pointerevents/pointerevent_constructor.html

```diff
@@ -30,6 +30,8 @@
             var testHeight = 5;
             var testTiltX = -45;
             var testTiltY = 30;
+            var testButton = 0;
+            var testButtons = 1;
```

Comment:
I don't have a strong opinion either way, but developers shouldn't be creating events with values that can't exist for real.